src/main/java/com/ernie/TicketApp/Ticket.java

```diff
@@ -0,0 +1,134 @@
+package com.ernie.TicketApp;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+public class Ticket extends Entity {
+
+    @NullableWarning
+    private UUID id = UUID.randomUUID();
```

Comment:
this field will be already defined from Entity constructor, you do not need to declare it in subclass additionally

Reply:
done in commit ba78624df638ea7b420b962d243bdefb20792f8d